package.json

```diff
@@ -1,37 +1,49 @@
 {
```

Comment:
It looks like `tippy.js` is no longer required. Please revert this file.

Reply:
Please revert your changes to package.json

Reply:
I have revert the changes in package.json file if still there is an issue then please elaborated the issue I am not able to understand what to change in the package.json file.

Reply:
I think you need to run a `git revert` so that this file doesn't show up on your PR. I believe the steps you need to follow are:

1) Navigate to your local blockly-samples directory via the command line.
2) Run `git checkout b304a7a -- package.json`. Where b304a7a is the current HEAD commit for the main repo's master branch. See [here](https://github.com/google/blockly-samples/commits/master).
3) Commit the reversion.

But I haven't done a reversion like that in a long time! Usually I just revert whole commits hehe. But I think that should be right.